src/screens/Exchange/Exchange.js

```diff
@@ -716,13 +778,36 @@ class ExchangeScreen extends React.Component<Props, State> {
   generateAssetsOptions = (assets) => {
     const { balances, paymentNetworkBalances } = this.props;
     const assetsList = Object.keys(assets).map((key: string) => assets[key]);
-    const nonEmptyAssets = assetsList.filter((asset: any) => {
-      return getBalance(balances, asset.symbol) !== 0 || asset.symbol === ETH;
+    const fiatCurrencies = [
+      {
+        symbol: defaultFiatCurrency,
+        decimals: 2,
+        iconUrl: 'asset/images/fiat/gbpColor.png',
+        iconMonoUrl: 'asset/images/fiat/gbpColor.png',
+      },
+      {
+        symbol: USD,
+        decimals: 2,
+        iconUrl: 'asset/images/fiat/usdColor.png',
+        iconMonoUrl: 'asset/images/fiat/usdColor.png',
+      },
+      {
+        symbol: EUR,
+        decimals: 2,
+        iconUrl: 'asset/images/fiat/eurColor.png',
+        iconMonoUrl: 'asset/images/fiat/eurColor.png',
+      },
+    ];
+    const cryptoFiatAssets = assetsList.concat(fiatCurrencies);
+    const nonEmptyAssets = cryptoFiatAssets.filter((asset: any) => {
+      return getBalance(balances, asset.symbol) !== 0 || [ETH, USD, defaultFiatCurrency, EUR].includes(asset.symbol);
```

Comment:
I would extract this `[ETH, USD, defaultFiatCurrency, EUR].includes(asset.symbol)` as well, you're using it in a lot of places

Reply:
0291e7e, except this line because it had ETH check on the previous version